.travis.yml

```diff
@@ -1,19 +1,24 @@
 sudo: required
+
 services:
 - docker
-language: node_js
-node_js:
-- node
+
+before_install:
+  - sudo rm /usr/local/bin/docker-compose
+  - curl -L https://github.com/docker/compose/releases/download/${DOCKER_COMPOSE_VERSION}/docker-compose-`uname -s`-`uname -m` > docker-compose
+  - chmod +x docker-compose
+  - sudo mv docker-compose /usr/local/bin
+
 script:
-- npm run lint
-- npm run snyk-test
-- npm run generate-coverage
-- npm run check-coverage
-- npm run upload-coverage-coveralls
+  - docker-compose -f docker-compose-travis.yml up --build -d
```

Comment:
I've got the tests to run (by adding the volumes as we do on 'local') - this is why I was insisting that the local tests are as similar to the travis ones as possible (maybe including the coverage). I want to feel confident that I can run the same tests - maybe parameterise the `TRAVIS=true` env variable and use one test file for both local and travis?